app/services/populi_service.rb

```diff
@@ -5,8 +5,8 @@ class PopuliService
 
   def initialize
       PopuliAPI.connect(
-          url: ENV["POPULI_API_URL"],  
-          access_key: ENV["POPULI_API_ACCESS_KEY"]
+          url: ENV["POPULI_API_URL"] || "https://fake-populi-domain.com",  
+          access_key: ENV["POPULI_API_ACCESS_KEY"] || "abc123"
```

Comment:
It looks like there are some dummy placeholder ENV vars set in CircleCI for these: https://app.circleci.com/settings/project/github/turingschool/present/environment-variables?return-to=https%3A%2F%2Fapp.circleci.com%2Fpipelines%2Fgithub%2Fturingschool%2Fpresent

Having built-in defaults will make running the tests locally easier, but it's probably also worth logging it to help someone debugging if they don't realize that they're using the default dummy creds in their app and seeing generic 404 or auth errors.